src/network/networkd-link.h

```diff
@@ -210,6 +210,7 @@ DEFINE_TRIVIAL_CLEANUP_FUNC(Link*, link_unref);
 DEFINE_TRIVIAL_DESTRUCTOR(link_netlink_destroy_callback, Link, link_unref);
 
 int link_get(Manager *m, int ifindex, Link **ret);
```

Comment:
maybe remain this function to `link_get_by_ifindex()` and its backing hashmap to `links_by_ifindex`?